_literate/EX-powergen/Project.toml

```diff
@@ -3,6 +3,5 @@ CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 MLJ = "add582a8-e3ab-11e8-2d5e-e98b27df1bc7"
 MLJLinearModels = "6ee0df7b-362f-4a72-a706-9e79364fb692"
-PyPlot = "d330b81b-6aea-500a-939a-2ce795aea3ee"
```

Comment:
Don't you need Plots here?

Reply:
Okay. I am 100% certain that I viewed all plots I fixed here (and elsewhere!) and I really can't explain why no error was thrown.

Reply:
If I do `using Plots` then activate an environment without `Plots` then is it possible that I can run plotting code?

Reply:
Yes! Changing your environment has not effect on loaded code. Generally, only do this if you really know what your doing, and never in final testing.

Reply:
Thank you. I should have expcted that.